R/shared-methods.R

```diff
@@ -173,10 +178,17 @@ DataClass <- R6::R6Class(
     #' cleaning steps be kept and output in a list.
     #' @param get Logical, defaults to FALSE. Should the class `get` method be
     #' called (this will download, clean, and process data at initialisation).
+    #' @param process_fns array, additional functions to process the data.
+    #' Users can supply their own functions here which would act on clean data
+    #' and they will be called alongside our default processing functions.
```

Comment:
maybe add a docs note that we want user submissions of new processing functions?